pkg/binfmt/Makefile

```diff
@@ -14,7 +15,8 @@ hash: $(DEPS)
 
 tag: hash
 	docker pull linuxkit/$(IMAGE):$(shell cat hash) || \
```

Comment:
this should set DCT at least in the case we are doing `sign` else if there is an existing unsigned tag it wont be replaced. Maybe means splitting signed and unsigend cases though.